go/chat/signencrypt/codec.go

```diff
@@ -157,10 +156,11 @@ func makeChunkNonce(nonce Nonce, chunkNum uint64) SecretboxNonce {
 	return &ret
 }
 
-func makeSignatureInput(plaintext []byte, encKey SecretboxKey, chunkNonce SecretboxNonce) []byte {
+func makeSignatureInput(plaintext []byte, encKey SecretboxKey, signaturePrefix libkb.SignaturePrefix, chunkNonce SecretboxNonce) []byte {
 	chunkHash := sha512.Sum512(plaintext)
 	var ret []byte
-	ret = append(ret, SignaturePrefix...)
+	ret = append(ret, signaturePrefix...)
+	ret = append(ret, 0x00)
```

Comment:
Let's add a short comment here just to clarify why this is important. There's a link about it in the big comment at the top of the file.